src/tasks/ts/safe.ts

```diff
@@ -6,22 +6,28 @@ import {
   MetaTransaction,
 } from "@gnosis.pm/safe-contracts";
 import GnosisSafe from "@gnosis.pm/safe-contracts/build/artifacts/contracts/GnosisSafe.sol/GnosisSafe.json";
-import GnosisSafeProxyFactory from "@gnosis.pm/safe-contracts/build/artifacts/contracts/proxies/GnosisSafeProxyFactory.sol/GnosisSafeProxyFactory.json";
+import CompatibilityFallbackHandlerDeployment from "@gnosis.pm/safe-deployments/src/assets/v1.3.0/compatibility_fallback_handler.json";
```

Comment:
Was there any official requirement for the fallback handler? I can not remember any discussion.

Reply:
There was no official requirement. I was discussing with Auryn and he mentioned that this is needed for receiving ERC1155 tokens, which I thought was good to have. Will start a discussion in #dex-token.